cypress/e2e/clue/branch/student_tests/dataflow_tool_spec.js

```diff
@@ -35,6 +35,18 @@ context('Dataflow Tool Tile', function () {
         dataflowToolTile.getNumberField().type("3{enter}");
         dataflowToolTile.getNumberField().should("have.value", "3");
       });
+
+      //TO DO: write a test that can check min and max (should be 0 and 3)
```

Comment:
So I tried a bit to target the class` chartjs-size-monitor` which I feel like is where ChartJs abstracts all of the chart code. I could get it to prove it exists but tried a few cypress classes with to search for max 3 or min 0 and no luck. 
I can't even really expand it in the DOM (it instantly closes), I left a ToDo statement in the cypress test, maybe there's another way.  I just added two tests to click the positive and negative buttons.

Reply:
Let me know if there's other tests you think are possible, because if we can't test for the values held in `tickMin` and `tickMax` I can't really test the `resetGraph` function we wrote.

Reply:
I just spent a few minutes looking as well, and I agree, it looks pretty difficult to add tests for the graphs. Thanks for taking the time to check.